src/modules/Builder/components/ConditionRow/ConditionRow.utils.ts

```diff
@@ -53,7 +53,7 @@ export const getItemOptions = (items: ItemFormValues[], conditionRowType: Condit
     }
 
     return optionList;
-  }, []);
+  }, []) || [];
```

Comment:
Is not it enough to have initialValue in `reduce`?

Reply:
if items undefined, then items?.reduce(...)  reduce is not launching, and function returns undefined